### PR TITLE
Timur katimullin 81 90 tasks

### PR DESCRIPTION
**Pull request by linkouth**
Adds 81-90 tasks solutions.

### CODE REVIEW

File: tasks/82_bouncing-balls/bouncing-balls.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+ * @example
+ * // return 15
+ * bouncingBall(30.0, 0.66, 1.5);
+ * @param {Number} h - the height of the floor
+ * @param {Number} bounce - the ball bounces for (bounce * h) metres
+ * @param {Number} window - the height of the window
+ * @return {Number}
+ */
+export function bouncingBall(h, bounce, window) {
+  let times = Math.floor(Math.log(window / h) / Math.log(bounce));
```
Comment (vovcyan) [inline tasks/82_bouncing-balls/bouncing-balls.js]: const

File: tasks/86_working-hours/working-hours.js [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+ * // 'SAT: 10:00 - 23:00',
+ * // 'SUN: 11:00 - 23:00'
+ * // ]
+ * formatWorkingHours(INPUT_DATA);
+ * @param {Array<Object>} days
+ * @return {Array<String>}
+ */
+export function formatWorkingHours(days) {
+  const schedule = days.reduce((schedule, day) => {
+    schedule[day.day.toUpperCase()] = day.from + ' - ' + day.to;
```
Comment (vovcyan) [inline tasks/86_working-hours/working-hours.js]: Split it

File: tasks/86_working-hours/working-hours.js [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+    return schedule;
+  }, {});
+
+  const weekByDays = ['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN']
+      .map((day) => ({
+        dayStart: day,
+        dayEnd: '',
+        hoursRange: schedule[day],
+      }));
+  const result = [weekByDays[0]];
```
Comment (vovcyan) [inline tasks/86_working-hours/working-hours.js]: Use ES6 features

File: tasks/86_working-hours/working-hours.js [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+      result.push(weekByDays[i]);
+    }
+  }
+  return result
+      .filter((item) => item.hoursRange)
+      .map((day) => {
+        const dayEnd = day.dayEnd === '' ? '' : ' - ' + day.dayEnd;
+        return day.dayStart + dayEnd + ': ' + day.hoursRange;
+      }
+      );
```
Comment (vovcyan) [inline tasks/86_working-hours/working-hours.js]: Move it to previous line

**Review by vovcyan — CHANGES_REQUESTED** [review]

**Review by akimovaa — APPROVED** [review]

**Review by KalachevDev — APPROVED** [review]

**Pull request merged**